src/lambda.ts

```diff
@@ -191,7 +191,7 @@ async function buildMountpointFromZipfile (zipfile: string, mountpointParent?: s
 
     await new Promise((resolve, reject) => {
       const endOnError = (error: Error) => reject(error);
-      unzipper.on('close', () => resolve());
+      unzipper.on('close', resolve);
```

Comment:
I'm not sure, but I thought I've had issues passing just the function reference in the past.

Reply:
@DavidTanner Yeah I probably pushed this PR a bit premature, a few tests failed, Im updating the call to just resolve as `undefined` instead. Should be the same result, Ill wait for the test suite then push up the update.